src/x265.mk

```diff
@@ -0,0 +1,74 @@
+# This file is part of MXE.
+# See index.html for further information.
+
+PKG             := x265
+$(PKG)_IGNORE   :=
```

Comment:
add WEBSITE and use single line `# This file is part of MXE. See LICENSE.md for licensing information.`